credoai/evaluators/performance.py

```diff
@@ -57,11 +58,6 @@ def __init__(self, metrics=None):
         self.prob_metrics = None
         self.failed_metrics = None
 
-    def _validate_arguments(self):
-        check_existence(self.metrics, "metrics")
-        check_data_instance(self.assessment_data, TabularData)
-        check_artifact_for_nulls(self.assessment_data, "Data")
-
```

Comment:
The evaluator schema I formalized in the docs is:
- init()
- validate()
- setup()

This mirrors what happens in the __call__() method. Can you move validate up before setup?